### PR TITLE
Install changes

### PR DESCRIPTION
**Pull request by fredrikhgrelland**

### CODE REVIEW

File: template/README.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ If you found this in `fredrikhgrelland/vagrant-hashistack`, you may be intereste
 
 Documentation on [parent repository](https://github.com/fredrikhgrelland/vagrant-hashistack#usage).
 
+## Prerequisites
+
+To run this box you will need Make, Vagrant, Virtualbox.
```
Comment (zhenik) [inline template/README.md:16]: `git cli` also ?

**Review by zhenik — CHANGES_REQUESTED** [review]

File: template/README.md (hunk 1 above)
Comment (fredrikhgrelland) [reply template/README.md:16]: Nah.. du har jo klonet repo?

Comment (zhenik) [reply template/README.md:16]: fair enough

**Review by zhenik — APPROVED** [review]

File: install/.env [hunk 2]
```diff
@@ -1,4 +1,4 @@
 VAGRANT_VERSION=2.2.9
-PACKER_VERSION=1.6.0
-VIRTUALBOX_VERSION=6.1.10
-VIRTUALBOX_FULLNAME=virtualbox-6.1_6.1.10-138449~Ubuntu~bionic_amd64.deb
+PACKER_VERSION=1.6.1
+VIRTUALBOX_VERSION=6.1.12
+VIRTUALBOX_FULLNAME=virtualbox-6.1_6.1.12-139181~Ubuntu~bionic_amd64.deb
```
Comment (Neha-Sinha2305) [inline install/.env:4]: @fredrikhgrelland 
Thinking out loud to myself: These binaries are installed when the user runs the **make install** command and most probably he would execute this command only once (the first time he sets up the whole environment). My question is :
1. Do we need to enforce the user to run the 'make install' command every time we upgrade the versions in this file?
2. What kind of issues the user might run into if they keep using the old binaries for say vagrant or the others that are included here?

Comment (fredrikhgrelland) [reply install/.env:4]: I am not sure... Please add this as an issue for later

**Pull request merged**